src/librbd/ImageCtx.cc

```diff
@@ -930,7 +930,8 @@ struct C_InvalidateCache : public Context {
         "rbd_journal_object_flush_age", false)(
         "rbd_journal_pool", false)(
         "rbd_journal_max_payload_bytes", false)(
-        "rbd_journal_max_concurrent_object_sets", false);
+        "rbd_journal_max_concurrent_object_sets", false)(
+        "rbd_mirror_resync_after_disconnect", false);
```

Comment:
It is not that I insists, though I can imagine a situation when you want the most of the images not to resync after disconnect (in order not to increase the load) while still have a group of "important" images you want to resync ASAP. Setting this option for these images via image-meta makes this possible. Also it simplifies testing -- no need to restart daemon to apply the option change.

@dillaman If you think these arguments are not enough to change your opinion I will remove this.

More logically I think it would be to have something like `rbd mirror image set $image resync_after_disconnect true`, but I guess it is not worth the effort, at least on this stage...